libraries/cli/include/cli/config_jsons/default.json

```diff
@@ -143,7 +144,6 @@
     ]
   },
   "chain_config": {
-    "chain_id": "0x4",
```

Comment:
hmm chain_id sounds like a chain_config param ?

Reply:
as we unified network_id and chain_id it can not be chain config param anymore

Reply:
and one more thing chain config is only optional